pkg/collector/prometheus_collector.go

```diff
@@ -382,6 +404,32 @@ func (p *PrometheusCollector) newContainerMetrics() {
 		[]string{"pod_name", "container_name", "container_namespace", "command"}, nil,
 	)
 
+	// cGroups Counters (counter)
+	containerCgroupCPUUsageUsTotal := prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "container", "cgroupfs_cpu_usage_us_total"),
+		"Aggregated cpu usage obtain from cGroups",
```

Comment:
`obtained from` (and in the following metrics names)

Reply:
rebased and amended the changes.
https://github.com/sustainable-computing-io/kepler/compare/376ae12c5195f881523915210823de52655e4067..a33e879567950b9933efbcaca0191a0f0a6de391
https://github.com/sustainable-computing-io/kepler/compare/a33e879567950b9933efbcaca0191a0f0a6de391..539d0000e8614c2d7b484fe83553160c18533c60